config/routes.rb

```diff
@@ -9,4 +9,6 @@
   get '/admin', to: 'dashboard#index'
 
   get '*path', to: redirect('/')
+
+  post '/update_youtube/:id', to: 'episodes#update_youtube'
```

Comment:
How can you update [Youtube](https://www.youtube.com/)?

Reply:
edited naming